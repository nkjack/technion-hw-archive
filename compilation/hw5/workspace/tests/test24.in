//function calls
void foo() {
	print("foo!\n");
}

void bar(int i) {
	printi(i);
	print("\n");
}

void moo(bool cond) {
	if (cond)
		foo();
	else
		bar(110);
}

void main() {
	foo();
	bar(150);
	moo(true);
	foo();
	moo(false);
}