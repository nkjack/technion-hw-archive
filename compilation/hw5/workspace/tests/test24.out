Loaded: ./exceptions.s
foo!
150
foo!
foo!
110
